packages/react-hooks/src/hooks/tests/media.test.tsx

```diff
@@ -2,13 +2,7 @@ import React from 'react';
 import {matchMedia, mediaQueryList} from '@shopify/jest-dom-mocks';
 import {mount} from '@shopify/react-testing';
 
-import {useMedia} from '../media';
-
-function MockComponent({mediaQuery}: {mediaQuery: string}) {
-  const matchedQuery = useMedia(mediaQuery);
-  const message = matchedQuery ? 'matched' : 'did not match';
-  return <div>{message}</div>;
-}
+import {useMedia, useMediaLayout} from '../media';
 
 describe('useMedia', () => {
```

Comment:
1. Could you rename to `'useMedia and useMediaLayout'`?
2. Also you could use `describe.each` instead of doing a `forEach` on line 16

Reply:
Done! Thank you! TIL about `describe.each`.